server/api/services/convertAddress.service.ts

```diff
@@ -0,0 +1,26 @@
+import axios from 'axios';
+
+class Geocode {
+  convertAddress = async (address: string | undefined) => {
+    if (address !== undefined) {
```

Comment:
Avoid huge blocks. Rearrange the code so it returns `{}` early if it is undefined.